doc/source/plans.rst

```diff
@@ -150,6 +181,46 @@ pseudo-axis. It's all the same to the plans. Examples:
     RE.subscribe(bec)
     RE(scan([det], motor, 1, 5, 5))
 
+.. note::
+
+   Why don't scans have ``delay`` parameter?
+
+   You may have noticed that :func:`count` has a ``delay`` parameter but none
+   of the scans do. This is intentional.
+
+   The common reason for wanting a delay in a scan is to allow a motor to
+   settle or a temperature controller to reach equilibrium. It is better to
+   configure this on the respective devices, so that scans will always add the
+   appropriate delay for the particular device being scanned.
+
+   .. code-block:: python
+
+      motor.settle_time = 1
+      temperature_controller.settle_time = 10
+
+   For many cases, this is more convenient and more robust than typing a delay
+   parameter in every invocation of the scan. You only have to set it once, and
+   it applies thereafter.
+
+   This is why bluesky leaves ``delay`` out of the scans, to guide users toward
```

Comment:
Checking my English, why not `towards`?
```suggestion
   This is why bluesky leaves ``delay`` out of the scans, to guide users towards
```

Reply:
Good question. I looked this up in [Garner's Modern American Usage](https://www.amazon.com/Garners-Modern-American-Usage-Garner/dp/0195382757) to check myself. He advises:

> In American English, the preferred form is *toward*; *towards* is prevalent in British English.

Reply:
Ah, got it! I learned British English at school 🇬🇧